java/client/src/org/openqa/selenium/remote/tracing/EventAttribute.java

```diff
@@ -34,6 +34,22 @@ public static EventAttributeValue setValue(long value) {
     return new EventAttributeValue(value);
   }
 
+  public static EventAttributeValue setValue(String[] value) {
```

Comment:
Why not use varargs here? That makes usage more like `setValue("foo", "bar")` and ends up using an array behind the scenes.

Reply:
+1  Will update it.